app/views/shared/_header.html.erb

```diff
@@ -43,10 +43,10 @@
         <li class="<%= 'active' if current_page?(controller: '/deploys', action: 'recent') %>">
           <%= link_to "Recent Deploys", recent_deploys_path %>
         </li>
-        <li class="<%= 'active' if current_page?(controller: '/deploys', action: 'active') %>">
+        <li class="current-deploys <%= 'active' if current_page?(controller: '/deploys', action: 'active') %>">
           <%= link_to active_deploys_path do %>
             Current Deploys
-            <%= content_tag(:span, Deploy.active.count, class: "badge") unless Deploy.active.count.zero? %>
+            <%= content_tag(:span, Deploy.active.count, class: "badge badge-deploys", style: "display: none;", data: { refresh_url: active_deploys_url(format: 'json') })  %>
```

Comment:
A preferred way is to use `hide` class (from bootstrap I assume) instead of inline style `display: none;`